locales/en/messages.json

```diff
@@ -971,6 +971,10 @@
         "message": "One of the others disarm flags is active when arming",
         "description": "Message that pops up to describe the ARM_SWITCH arming disable flag"
     },
+    "initialSetupArmingDisableFlagsTooltipMOTOR_PROTO": {
+        "message": "There is no motor output protocol selected",
+        "description": "Message that pops up to describe the ARM_SWITCH arming disable flag"
```

Comment:
Good catch! But the description is not correct 😊

And we try to keep the same order than the firmware, I think the disarm flag is the latest one always.

Reply:
fixed now

Reply:
now it's fixed